src/test/java/com/jcabi/log/MulticolorLayoutTest.java

```diff
@@ -41,6 +41,19 @@
  * Test case for {@link MulticolorLayout}.
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
+ * @todo #54:30min Create an integration test for testing if MulticolorLayout
+ *  will disable text coloring when an application is run with
+ *  `-Dcom.jcabi.log.coloring=false`.
+ *  ...
+ *  Intergration testing is required because this can't be done in a unit test:
+ *  ...
+ *  - We can't just pass a parameter to `MulticolorLayout` that determines if
+ *  the layout should color its output, because Log4J requires
+ *  EnhancedPatternLayout to have a 0-arguments constructor.
+ *  - We can't disable coloring with something like
+ *  `MulticolorLayout#setColoring(boolean)`, too, (that would be unit-testable)
+ *  because Log4J won't know how to call that.
+ *  - `System.getProperty(String)` is not mockable for obvious reasons.
```

Comment:
Here is a log for this PR: https://s3.amazonaws.com/archive.travis-ci.org/jobs/75081161/log.txt
You can search for `threadCount`, it says 4. Apart from that, I tried testing this feature, and tests ran fine with "Run tests" in IntelliJ IDEA (runs in a single thread), but failed in `mvn test` with output that indicates that the system property was indeed set to a wrong value.